packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -997,14 +997,14 @@ export const Transaction = React.memo(function Transaction(props) {
         name="debit"
         exposed={focusedField === 'debit'}
         focused={focusedField === 'debit'}
-        value={debit == null ? '' : debit}
```

Comment:
The `debit` and `credit` fields come from `serializeTransaction()`. If you look at how it works.. it will NEVER return these two fields as "null". Best it can do - it returns an empty string.

So fixed this check to show "0.00" value if both debit & credit is blank.